irondb/release-notes.md

```diff
@@ -5,6 +5,62 @@ sidebar_position: 12
+   for details.
+ * Fix bug where the `reconst_in_progress` file was not being cleaned up after
+   reconstitute operations, which could block rollups and deletes from running.
+ * The `raw/rollup` and `histogram_raw/rollup` API endpoints will no longer
+   block if there is a rollup already running. They will also return sensible
+   error messages.
+ * Raw shard rollups will not be allowed to run unless all previous rollups
+   have run at least once.
+ * Fix bug where deferred rollups could cause the rollup process to lock up
+   until the node is restarted.
+ * Add REST endpoint POST/PUT
+   `/histogram/<period>/<check_uuid>/<metric_name>?num_records=<x>` which can
+   be used with a json payload to directly insert histogram shard metric data
+   for repair purposes.
+ * Added configuration file field, `//reconstitute/@max_simultaneous_nodes`,
+   that will cause the reconstituting node to only hit a predetermined
+   number of peer nodes at once. The default if not specified is "all peers".
+   This setting can be used if a reconstitute places too much load on the rest
+   of the cluster, causing degradation of service.
+ * Disallow starting single-shard reconstitutes with merge enabled if the shard
+   exists and is flagged corrupt.
+ * Improve NNTBS error messages if unable to open a shard.
+ * PromQL - improve error messages on invalid or unsupported range queries.
+ * PromQL - fix range nested inside one or more instant functions.
+ * Include maintenance mode when pulling lists of raw, histogram, or histogram
+   rollup shards.
+ * Use read-copy-update (RCU) for Graphite level indexes and the surrogate
+   database. It allows more queries to run concurrently without affecting
+   ingestion, and vice versa.
+ * Defer rollups of raw shards if there is a rollup shard in maintenance that
+   the raw shard would write to.
+ * Reject live shard reconstitute requests on NNTBS or histogram rollup shards
+   if there is a raw shard rollup in progress that would feed into them.
+ * Fix bug where the system would report that a live reconstitute was not in
+   progress, even when one was running.
+ * Allow running single-shard or single-metric reconstitute on non-raw shards,
+   even if the shard extends beyond the current time.
+ * The reconstitute GUI no longer apppears when doing online reconstitutes.
+ * Fix iteration bug when reconstituting NNTBS shards.
+ * Added the `merge_all_nodes` flag to live reconstitute which causes all
+   available and non-blacklisted write copies to send metric data instead of
+   only the "primary" available node.
+ * Added the ability to repair a local database by reconstituting a single
+   metric stream.
+ * Fix bug where `/fetch` would not proxy if the data for a time period was all
+   in the raw database, but the relevant raw shards were offline.
+ * [libmtev 2.5.3](https://github.com/circonus-labs/libmtev/blob/master/ChangeLog.md#253)
```

Comment:
```suggestion
```

Reply:
This was already in the prior release, don't think we need to add it again

Reply:
Why remove this?

Reply:
Nevermind, I see that we got 2.5.3 with 1.0.1.